domain-models-api-aspects/src/main/java/org/folio/rest/validators/mapper/CustomValidationExceptionMapper.java

```diff
@@ -0,0 +1,22 @@
+package org.folio.rest.validators.mapper;
+
+import javax.validation.ValidationException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import org.apache.log4j.Logger;
+
+@Provider
+public class CustomValidationExceptionMapper implements ExceptionMapper<ValidationException> {
+
+  private static final Logger log = Logger.getLogger(CustomValidationExceptionMapper.class);
+
+  @Override
+  public Response toResponse(ValidationException exception) {
+    log.info("Handling validation exception: " + exception.getMessage());
+    log.error("Validation Exception", exception.getCause());
```

Comment:
If loglevel is error then only the stacktrace but not exception.getMessage() gets logged. Why do you want to suppress the message?

Can you join the two log.* lines into a single log.error(...) invokation?

Reply:
Example invokation:
```
    Exception exception = new RuntimeException("Some detail.");
    log.info("Handling validation exception: " + exception.getMessage());
    log.error("Validation Exception", exception.getCause());
```
Output with loglevel set to info:
```
28 Aug 2018 11:57:25:238 INFO  Log4jTest  Handling validation exception: Some detail.
28 Aug 2018 11:57:25:251 ERROR Log4jTest  Validation Exception
```
Output with loglevel set to warn:
```
28 Aug 2018 11:53:27:354 ERROR Log4jTest  Validation Exception
```
The string "Some detail" is missing from the log.

Reply:
Thanks for the example. To be honest, it was a cut and paste. I would of guessed the cause would include the message. Anyhow, the entire class is not needed for this feature.